lib/pnpm.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+YQ="$BP_DIR/lib/vendor/yq-$(get_os)"
+
+detect_pnpm() {
+  echo "true"
```

Comment:
Do we need this script? I can't see where `YQ` is used, and can only see it being set in the `yarn2.sh` file currenlty

Reply:
I guess not, I have now deleted it 👍 .